Copying Github templates from T4Rec (#165)
NVIDIA-Merlin/models: .github/ISSUE_TEMPLATE/feature-request.md

---
name: "\U0001F680 Feature request"

> about: Submit a proposal/request for a new Merlin Modelds feature

Reviewer: ```suggestion
about: Submit a proposal/request for a new Merlin Models feature
```

> about: Submit a proposal/request for a new Transformers4Rec feature

Reviewer: ```suggestion
about: Submit a proposal/request for a new Merlin Modelds feature
```

title: "[FEA]"
labels: "status/needs-triage, kind/feature-request"
assignees: ''

---

# 🚀 Feature request

<!-- A clear and concise description of the feature proposal.
     Please provide a link to the paper and code in case they exist. -->

## Motivation

<!-- Please outline the motivation for the proposal. Is your feature request
     related to a problem? e.g., I'm always frustrated when [...]. If this is related
     to another GitHub issue, please link here too. -->

## Your contribution

<!-- Is there any way that you could help, e.g. by submitting a PR?
     Make sure to read the CONTRIBUTING.MD readme:

>      https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/master/CONTRIBUTING.md -->

Reviewer: ```suggestion
     https://github.com/NVIDIA-Merlin/models/blob/main/CONTRIBUTING.md -->
```
